pom.xml

```diff
@@ -273,6 +280,11 @@
             <artifactId>mule-core</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mule.runtime</groupId>
+            <artifactId>mule-module-spring-config</artifactId>
```

Comment:
I think this is already added by a commit that was merged today.

Reply:
true.. removed